src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/Inputs/JSON1006.ValueTooLarge_LocationId_Invalid.sarif

```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.5.json",
+  "version": "2.1.0",
+  "runs": [
+    {
+      "tool": {
+        "driver": {
+          "name": "Sarif Functional Testing",
+          "informationUri": "https://www.example.com",
+          "version": "1.0"
+        }
+      },
+      "results": [
+        {
+          "ruleId": "TEST1001",
+          "message": {
+            "text": "Locations that specify file paths."
+          },
+          "locations": [
+            {
+              "id": 2147483648,
+              "physicalLocation": {
+                "artifactLocation": {
+                  "uri": "Test.Int32.Max+1.cs"
+                }
+              }
+            },
+            {
+              "id": 9223372036854775807,
+              "physicalLocation": {
+                "artifactLocation": {
+                  "uri": "Test.Int64.Max.cs"
```

Comment:
>Test.Int64.Max [](http://example.com/codeflow?start=26&length=14)

I use this as a place to add comments of the current test case.

Want to point out another issue,
In our other repo which is used here to do the JSON validations
[https://github.com/microsoft/jschema/blob/main/src/Json.Schema.Validation/Validator.cs](https://github.com/microsoft/jschema/blob/main/src/Json.Schema.Validation/Validator.cs)
in the whole solution we choose to use type `long` and `double`.
(in this case downsize from `long` to `int`)
This means field value should not exceed the range of `long` and `double`.
technically we can choose to replace `long` and `double` with larger range types,
but not sure if it is necessary,
and whatever range we increase to, user will still be able to add more digits to 
exceed the range, unless we make fields as string we can not prevent that.
I think the code choose to use `long` and `double` was a design choice not a bug, let me know.